src/main/java/frc/robot/commands/vision/AutonBallGetter.java

```diff
@@ -57,13 +63,13 @@ public void execute() {
             return;
         }
         /** Some math to find the real, actual angle of the blob from the camera. */
-        ballAngle = bloberooX[0] * (Const.AutonBallGetter.CAMERA_ANGLE_X / Const.AutonBallGetter.CAMERA_RES_X);
+        ballAngle = bloberooX[0] * (Const.AutonBallGetter.cameraAngleX / Const.AutonBallGetter.cameraResX);
         angleDiff = gyroAngle - ballAngle;
-        if ((angleDiff <= Const.AutonBallGetter.ACCEPTED_OFFSET_BOUNDS) && (angleDiff >= -Const.AutonBallGetter.ACCEPTED_OFFSET_BOUNDS)) {
+        if ((angleDiff <= Const.AutonBallGetter.ballGetterAcceptedOffsetBounds) && (angleDiff >= -Const.AutonBallGetter.ballGetterAcceptedOffsetBounds)) {
             angleDiff = 0;
         }
         /** Actually drive with the vector made from calculations. */
-        SwerveVector trackWithZ = new SwerveVector(0, 0, (angleDiff * Const.AutonBallGetter.BALL_ADJUST_SPEED));
+        SwerveVector trackWithZ = new SwerveVector(0, 0, (angleDiff * Const.AutonBallGetter.adjustSpeed));
         drivetrain.drive(trackWithZ);
```

Comment:
Again, be careful where you're pulling your variables from. You didn't add variables to the `Const` class; you added them to the top of this class.